app/src/main/java/org/kiwix/kiwixmobile/webserver/WebServerHelper.java

```diff
@@ -0,0 +1,66 @@
+package org.kiwix.kiwixmobile.webserver;
+
+import android.util.Log;
+import androidx.annotation.NonNull;
+import java.util.ArrayList;
+import javax.inject.Inject;
+import org.kiwix.kiwixlib.JNIKiwixException;
+import org.kiwix.kiwixlib.JNIKiwixLibrary;
+import org.kiwix.kiwixlib.JNIKiwixServer;
+import org.kiwix.kiwixmobile.utils.ServerUtils;
+
+/**
```

Comment:
If this comment is true shouldn't the polling for ip Address happen here?

Reply:
Fixed 1afbc5c6b1c1c42e1318c7e6689ab9e4feb76f59